frontend/app/components/atoms/drop-down/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import styles from "../../../../styles/Dropdown.module.scss";
+
+interface DropdownProps {
+  isVisible: boolean;
+}
+
+const locations = [
```

Comment:
these consts (locations, meetings, accounts) should be passed in as props to the dropdown component